docs/pages/verifiable-presentations.md

```diff
@@ -0,0 +1,280 @@
+# Verifiable Presentations (V1)
+
+This document describes how to create v1 verifiable presentations and how to verify them.
+
+**Table of Contents:**
+<!--toc:start-->
+- [Build Statement](#build-statement)
+  - [Identity/account credential statements](#identityaccount-credential-statements)
+  - [Web3 ID credential statements](#web3-id-credential-statements)
+- [JSON representation](#json-representation)
+- [Verifiable Presentation Request (proof request)](#verifiable-presentation-request-proof-request)
+- [Verifiable Presentation (proof)](#verifiable-presentation-proof)
+- [Verifiable Audit Record](#verifiable-audit-record)
+<!--toc:end-->
+
+## Build Statement
+
+The SDK contains a helper to create statements about identities, which can
+then be proven.
+
+To do so, use the CredentialStatementBuilder, to build a statement:
+
+{@codeblock ~~:nodejs/common/verifiable-credential-statements.ts#documentation-snippet}
+
+### Identity/account credential statements
+
+To build a statement against an identity credential, the builder has two different entrypoints, which
+have an identical function signature, which consists of
+
+1. A list of identity providers that the identity must be created from
+2. A callback function which should be used to add statements for the credential
+
+```ts
+// used for proofs which are not tied to a specific account
+builder.forIdentityCredentials([0,2], (build) => ...)
```

Comment:
It would be good to add code snippet to - import CredentialStatementBuilder class and create this `builder` object, I suppose?